src/config/environment.production.js

```diff
@@ -0,0 +1,16 @@
+class Env {
+  ApiBaseURL () { return 'https://eqip-prototype-api.fr.cloud.gov' }
```

Comment:
The Javascript is compiled and not sure we have access to environment variables in a static `.js` file.

Are we running this on a NodeJS instance? May have access to it then.